django-entrypoint.sh

```diff
@@ -1,63 +1,69 @@
 #!/usr/bin/env sh
 set -e
 
-if [ -z "$APP_MODULE" ]; then
-  echo "The \$APP_MODULE environment variable must be set to the WSGI application module name"
-  exit 1
+# No args or looks like options or the APP_MODULE for Gunicorn
+if [ "$#" = 0 ] || \
+    [ "${1#-}" != "$1" ] || \
+    echo "$1" | grep -Eq '^([_A-Za-z]\w*\.)*[_A-Za-z]\w*:[_A-Za-z]\w*$'; then
```

Comment:
I almost ran some more Python code to see if this described a module that could be imported 😿